### PR TITLE
inherit sale_view_adj_hls

### PR DESCRIPTION
**Pull request by SokreamPhanqrl**

### CODE REVIEW

File: sale_order_invoice_address_propose/views/sale_order_views.xml [hunk 1]
```diff
@@ -18,4 +18,15 @@
         </field>
     </record>
 
+    <record id="view_order_form_inherit_sale_view_adj_hls" model="ir.ui.view">
+        <field name="name">sale.order.form.inherit.sale.view.adj.hls</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale_view_adj_hls.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_invoice_id'][2]" position="attributes">
```
Comment (yostashiro) [inline sale_order_invoice_address_propose/views/sale_order_views.xml:23]: Is it necessary to have `[2]` here?

**Review by yostashiro — COMMENTED** [review]
@SokreamPhanqrl Can you please explain why you are not removing this part?  Just curious about the reasoning.
https://github.com/qrtl/hls-custom/blob/0d94072b8977ffc95f518d122b363f9097d6708d/sale_order_invoice_address_propose/views/sale_order_views.xml#L12-L14

File: sale_order_invoice_address_propose/views/sale_order_views.xml (hunk 1 above)
Comment (SokreamPhanqrl) [reply sale_order_invoice_address_propose/views/sale_order_views.xml:23]: Yes, without [2], it won't add the domain to partner_invoice_id.

**Comment by SokreamPhanqrl** [comment]
> @SokreamPhanqrl Can you please explain why you are not removing this part? Just curious about the reasoning.
> https://github.com/qrtl/hls-custom/blob/0d94072b8977ffc95f518d122b363f9097d6708d/sale_order_invoice_address_propose/views/sale_order_views.xml#L12-L14



> @SokreamPhanqrl Can you please explain why you are not removing this part? Just curious about the reasoning.
> https://github.com/qrtl/hls-custom/blob/0d94072b8977ffc95f518d122b363f9097d6708d/sale_order_invoice_address_propose/views/sale_order_views.xml#L12-L14
Probably, I forgot to check that part.

**Pull request merged**